### ISSUE TITLE
[Feature Request] Beta enrollment toggle

### ISSUE THREAD

**Issue by Spidy123222** [SideStore/SideStore#139]
The current idea I have of this that in SideStore the beta enrollment toggle which should be located in settings of the app will swap which default sidestore source it uses to use the beta branch json and will then have an update as it's the same bundleid and should work and change version. The name of the app can have a (beta) in the name of the app to tell between them first glance or have the tag system fixed and use that.

**Comment by Spidy123222** [comment]
I have tested that users can add a source that replaces the default source for that session and can induce updates.

**Comment by mahee96** [comment]
0.6.0 nightly has it and could be improved with release channels

**Comment by Aerozolic** [comment]
Just installed the latest alpha and the option to switch to beta and alpha disappeared. When installing nightly or alpha from the official repo it installs a second SideStore app.

**Comment by mahee96** [comment]
Yeah, we were experimenting things, and decided we are bringing back the in-app switching since separate app consumes appID…

**Comment by mahee96** [comment]
okay, for time being we reverted back to in-app release tracks switching so that nightly HotFixes can keep going. 

We are yet to decide on the spec for this in-app releaseTracks support or standalone app as beta releases.

**Comment by mahee96** [comment]
okay, we are keeping the in-app releaseTracks switching feature in #866
